docker-compose.yml

```diff
@@ -34,4 +34,16 @@ services:
     environment:
       POSTGRES_USER: "test"
       POSTGRES_PASSWORD: "test"
-      POSTGRES_DB: "test"
\ No newline at end of file
+      POSTGRES_DB: "test"
+
+  # mssql
+  mssql:
```

Comment:
There are 2 images one for Linux and windows. I am using the Linux image one on my windows machine and it's working fine. Seems to break on osx though even though it should technically work.